src/app/plugins/marketplace.plugin.ts

```diff
@@ -0,0 +1,70 @@
+import Constants from '../../common/constants';
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+import { RichEmbed } from 'discord.js';
+
+export class MarketPlacePlugin extends Plugin {
+  public name: string = 'MarketPlace';
+  public description: string = 'Stores and Lists Everything On MarketPlace.';
+  public usage: string = '!Market <add/list>';
+  public pluginAlias = [];
+  public permission: ChannelType = ChannelType.Public;
+  public pluginChannelName: string = Constants.Channels.Public.BuySellTrade;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args: string[]) {
+    if (!args) {
```

Comment:
if args = empty array, it will be truthy, we'll want to check `!args || args.length === 0`